code/zFulpstationCode/fulp_overwrite_vars.dm

```diff
@@ -446,3 +446,34 @@
 //**** Sleepy Pen for ops ENDS - Glubtok, Jul 2020
 //***********************************************************
 
+
+
+//***********************************************************
+//****UNSTAGRAV HOLODRAGON  PR *****************************
+//***********************************************************
+
+/obj/item/clothing/under/rank/engineering/atmospheric_technician
+	desc = "It's a jumpsuit worn by atmospheric technicians. It is protected from fire, not sure about you tho."
```

Comment:
```suggestion
	desc = "It's a jumpsuit worn by atmospheric technicians. While it can't catch on fire, the same might not necessarily be true for its wearer."
```